Charter.md

```diff
@@ -0,0 +1,19 @@
+## Node.js Testing Working Group
+
```

Comment:
Might want to add that this charter has not been ratified yet by the CTC, and Build WG (given the overlap with Build, it will probably need ratification by them too).